lib/gui/pages/main/controllers/image-selection.js

```diff
@@ -67,18 +66,6 @@ module.exports = function(
    *   .then(ImageSelectionController.selectImage);
    */
   this.selectImage = (image) => {
-    if (!supportedFormats.isSupportedImage(image.path)) {
-      const invalidImageError = errors.createUserError({
-        title: 'Invalid image',
-        description: messages.error.invalidImage({
-          image
-        })
-      });
-
-      OSDialogService.showError(invalidImageError);
-      AnalyticsService.logEvent('Invalid image', image);
```

Comment:
Even if we don't show an Error dialog, maybe it'd still be worth logging an "Unrecognised image format" event?

And instead of getting rid of the Error dialog completely, maybe we could/should change it to a "Warning" dialog, with a "Proceed anyway" button, like we do for Windows images?